### DISCUSSION TITLE
AL 2023 applicability for CVE-2021-43396 & CVE-2007-4559

### DISCUSSION THREAD

**Discussion by kusaurabh** [amazonlinux/amazon-linux-2022#721] [General]
Hi,

Our security scanner identifies the following CVEs  based on docker image amazoncorretto:17-al2023

CVE-2021-43396 
GNU C Library 2.34
glibc-common/2.34-52.amzn2023.0.7/x86_64
glibc/2.34-52.amzn2023.0.7/x86_64

CVE-2007-4559
Python programming language 3.9.16
python3-libs/3.9.16-1.amzn2023.0.6/x86_64
python3/3.9.16-1.amzn2023.0.6/x86_64

We tried to search for these in https://alas.aws.amazon.com/alas2023.html and https://explore.alas.aws.amazon.com/ but could not find any related advisory.

Please share if there are any security advisory from Al2023 security team on these.

Thanks

**Comment by alexey-tsvetnov** [comment]
@kusaurabh , this is a wrong place to report such issues.
This Github repo belongs to only Amazon Linux distribution.

Additionally, all mentioned CVEs were fixed:
- https://alas.aws.amazon.com/AL2023/ALAS2023-2025-988.html
- https://explore.alas.aws.amazon.com/CVE-2007-4559.html
